cosmos/spec/packets/structure_spec.rb

```diff
@@ -350,10 +350,10 @@ module Cosmos
     end
 
     describe "read_item", no_ext: true do
-      it "complains if no buffer given" do
+      it "works if no buffer given" do
```

Comment:
Because a buffer is created internally right?

Reply:
Yes